### ISSUE TITLE
Enhancement: Function to dump JSON responses to METADATA

### ISSUE THREAD

**Issue by astrochun** [ualibraries/LD_Cool_P#152]
**Summary**
Create a function to export JSON responses from the Figshare API and write to METADATA curation folder.

**Objectives**
As a starting point, have a `METADATA/curation.json` from the curation message. The format should be prettify for readability (e.g., using `pprint` or something similar).

**Proposal**

- [x] Create a module with a simple function for JSON export within the `curation` sub-package
- [x] Implement it in the `curation.main` step
    - [x] When retrieving curation information: Add `PrerequisiteWorkflow.write_curation_metadata` method
    - [x] Write list of files in `retrieve.download_files`

*Testing notes*
- [x] Test on a stage deposit
- [x] Test on a production deposit (tested with 12935162)

*Additional notes*
- [x] This should begin after #151 is addressed in `develop` branch

**Implemented in**: TBD

**Comment by astrochun** [comment]
Because #137 requires a bit of re-working on `Qualtrics`. I think it's best to hold off on storing metadata records. Will migrate those action items to a separate issue
